src/main/scala/com/cognite/sdk/scala/common/dataTypes.scala

```diff
@@ -4,19 +4,28 @@
 package com.cognite.sdk.scala.common
 
 import java.time.Instant
-
 import cats.Id
 import com.cognite.sdk.scala.v1.CogniteId
 import io.circe.{Decoder, Encoder, Json, JsonObject, KeyEncoder}
-import io.circe.generic.semiauto.deriveDecoder
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
 import sttp.model.Uri
 // scalastyle:off number.of.types
 trait ResponseWithCursor {
   val nextCursor: Option[String]
 }
 final case class ItemsWithCursor[A](items: Seq[A], nextCursor: Option[String] = None)
     extends ResponseWithCursor
+object ItemsWithCursor {
+  implicit def itemsWithCursorEncoder[A: Encoder]: Encoder[ItemsWithCursor[A]] =
+    deriveEncoder[ItemsWithCursor[A]]
+  implicit def itemsWithCursorDecoder[A: Decoder]: Decoder[ItemsWithCursor[A]] =
+    deriveDecoder[ItemsWithCursor[A]]
+}
 final case class Items[A](items: Seq[A])
+object Items {
+  implicit def itemsEncoder[A: Encoder]: Encoder[Items[A]] = deriveEncoder[Items[A]]
+  implicit def itemsDecoder[A: Decoder]: Decoder[Items[A]] = deriveDecoder[Items[A]]
+}
```

Comment:
No need to add encoders/decoders for `Items` and `ItemsWithCursor`.
They can be auto derived with this.